src/Service.js

```diff
@@ -1,5 +1,5 @@
- // const baseUrl = 'http://localhost:5000/';
- const baseUrl = "https://damp-citadel-25681.herokuapp.com/"
+ const baseUrl = 'http://localhost:5000/';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/CSC510-Group-25/feature-hunt/pullRequest?prid=8363319)
```suggestion
const baseUrl = "http://localhost:5000/";
```

Reply:
Fiiiine